src/test/scala/io/qbeast/spark/index/SparkRevisionChangesUtilsTest.scala

```diff
@@ -658,4 +661,60 @@ class SparkRevisionChangesUtilsTest
     revision_2.desiredCubeSize shouldBe 10000
   }
 
+  it should "createNewRevision with column stats for all numeric types" in withSpark(spark => {
```

Comment:
There's a list of tests at the beginning of this file. It was made for clarity and to organize the tests, as there are too many cases to cover. 

Please add this test to the list and place it in its proper place, i.e., somewhere in the section of _**// Transformation changes (with columnStats)**_